### ISSUE TITLE
Structure loading failed report from Blame

### ISSUE THREAD

**Issue by davidalb97** [yungnickyoung/YUNGs-Better-Dungeons#18]
I am creating this issue on the off-chance this is really bug and not a feature. 
Loading Yung's Better Dungeons with [Blame](https://github.com/TelepathicGrunt/Blame/tree/Fabric-1.17.x) will print the following error on the server console:

```
[Worker-Main-7/INFO]: Preparing spawn area: 16%
[Worker-Main-9/INFO]: Preparing spawn area: 17%
[Worker-Main-7/INFO]: Preparing spawn area: 20%
[Worker-Main-6/ERROR]: 
****************** Blame Report Structure NBT 1.17-4.2.0-fabric ******************

 Failed to load structure nbt file from : minecraft:2x3_3
 The path represented by this is:  data/minecraft/structures/2x3_3.nbt
 Most common cause is that there is a typo in this path to the nbt file.
 Please let the mod author know about this so they can double check to make sure the path is correct.
 A common mistake is putting the structure nbt file in the asset folder when it needs to go in data/structures folder.
  Stacktrace of where this NBT file is being called by:[
    java.base/java.lang.Thread.getStackTrace(Thread.java:1596), 
    com.telepathicgrunt.blame.main.MissingNBTBlame.addMissingNbtDetails(MissingNBTBlame.java:56), 
    net.minecraft.class_3485.handler$zcj000$blame_addMissingnbtDetails2(class_3485.java:535), 
    net.minecraft.class_3485.method_15094(class_3485.java:65), 
    net.minecraft.class_3485.method_15091(class_3485.java:53), 
    com.mojang.datafixers.util.Either$Left.map(Either.java:38), 
    net.minecraft.class_3781.method_27233(class_3781.java:84), 
    net.minecraft.class_3781.method_16627(class_3781.java:109), 
    com.yungnickyoung.minecraft.yungsapi.world.jigsaw.JigsawManager$Assembler.processList(JigsawManager.java:291), 
    com.yungnickyoung.minecraft.yungsapi.world.jigsaw.JigsawManager$Assembler.processPiece(JigsawManager.java:206), 
    com.yungnickyoung.minecraft.yungsapi.world.jigsaw.JigsawManager.assembleJigsawStructure(JigsawManager.java:107), 
    com.yungnickyoung.minecraft.yungsapi.api.YungJigsawManager.assembleJigsawStructure(YungJigsawManager.java:63), 
    com.yungnickyoung.minecraft.betterdungeons.world.structure.skeleton_dungeon.SkeletonDungeonStructure$Start.init(SkeletonDungeonStructure.java:80), 
    com.yungnickyoung.minecraft.betterdungeons.world.structure.skeleton_dungeon.SkeletonDungeonStructure$Start.method_16655(SkeletonDungeonStructure.java:57), 
    net.minecraft.class_3195.method_28657(class_3195.java:269), 
    net.minecraft.class_5312.method_28622(class_5312.java:36), 
    net.minecraft.class_2794.method_28508(class_2794.java:309), 
    net.minecraft.class_2794.method_16129(class_2794.java:292), 
    net.minecraft.class_2806.method_16556(class_2806.java:53), 
    net.minecraft.class_2806.method_12154(class_2806.java:231), 
    net.minecraft.class_3898.method_17225(class_3898.java:556), 
    com.mojang.datafixers.util.Either$Left.map(Either.java:38), 
    net.minecraft.class_3898.method_17224(class_3898.java:550), 
    java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1146), 
    java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478), 
    net.minecraft.class_3900.method_17634(class_3900.java:58), 
    net.minecraft.class_3846.method_16907(class_3846.java:91), 
    net.minecraft.class_3846.method_16900(class_3846.java:146), 
    net.minecraft.class_3846.run(class_3846.java:102), 
    java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1434), 
    java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295), 
    java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016), 
    java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665), 
    java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598), 
    java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
]
[Worker-Main-6/INFO]: Preparing spawn area: 20%
[Worker-Main-6/INFO]: Preparing spawn area: 22%
```

Modlist:
*  betterstrongholds@1.17-1.1.1
*  blame@1.17-4.2.0-fabric
*  fabric@0.40.1+1.17
*  yungsapi@1.17-Fabric-16

[log on pastebin](https://pastebin.com/Pxj2VNcv)

I can also confirm that this error does not happen on other mods like Yung's Better Strongholds with blame.

**Comment by yungnickyoung** [comment]
Looks like I forgot to namespace one of my pieces here: https://github.com/yungnickyoung/YUNGs-Better-Dungeons/blob/74e4abe1afa6a0825598887845e89b9b3c74033f/src/main/resources/data/betterdungeons/worldgen/template_pool/skeleton_dungeon/2x3.json#L35

It's a very minor bug that just results in a single type of prop in skeleton dungeons not being used, but good catch.

**Comment by davidalb97** [comment]
Would you like me to make 3 PRs with the change for each branch (forge/1.16, fabric/1.6, fabric/1.7), or you rather do it yourself?

**Comment by yungnickyoung** [comment]
Sure, if you don't mind!

**Comment by yungnickyoung** [comment]
Fixed in 1.18
